hipchat.go

```diff
@@ -218,4 +225,5 @@ func (c *Client) listen() {
 			}
 		}
 	}
+	panic("unreachable")
```

Comment:
i'm ok leaving it in. i just wanted to understand your reasoning. if that loop were to stop, it's `panic` worthy.